ocp_resources/job.py

```diff
@@ -1,10 +1,27 @@
+import kubernetes
+
 from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.resource import NamespacedResource
 
 
 class Job(NamespacedResource):
     """
     Job object.
+
+        Args:
+            name (str): Job name.
+            namespace (str): Namespace name.
+            client (DynamicClient): Dynamic client for connecting to a remote cluster.
+            teardown (bool): Indicates if this resource would need to be deleted.
+            privileged_client (DynamicClient): Instance of Dynamic client.
+            yaml_file (str): Yaml file for the resource.
+            delete_timeout (int): Timeout associated with delete action.
+            backoff_limit (int): The number of retries for a job.
+            restart_policy (str): The restart policy of the pod.
+            service_account (str): Optional. Service account name.
+            containers (list): List of containers belonging to the pod that the job will create.
+            background_propagation_policy (bool): If true - change propagation policy to 'Background'.
```

Comment:
`change propagation policy to 'Background'.` do we have more option other than `Background`? 
If so we need to get it here and not bool
Also please explain what is this and not just that we change it

Reply:
Ack.